### ISSUE TITLE
`unused-import` should observe `dummy-variables-rgx` setting

### ISSUE THREAD

**Issue by macdjord** [pylint-dev/pylint#8500]
### Bug description

Python convention is that any variable name consisting entirely of underscores is to be considered a placeholder, used when a value needs to be assigned somewhere but you don't actually care about it after that. Thus no warnings should be emitted about such a variable being unused, overwritten, or changing type.

However, another common convention is to import the `gettext()` translation function as `_` for convenience. Thus current best-practice is to use names of two or more underscores for placeholders.

`test2.py`:
```python
"""Test"""

import prettyprinter as __
```

### Command used

```shell
pylint test2.py
```

### Pylint output

```shell
************* Module test2
test2.py:3:0: W0611: Unused prettyprinter imported as __ (unused-import)

------------------------------------------------------------------
Your code has been rated at 0.00/10 (previous run: 0.00/10, +0.00)
```

### Expected behavior

No `unused-import` should be emitted if the module is imported with a name consisting entirely of underscores.

### Pylint version

```shell
pylint 2.16.2
astroid 2.14.2
Python 3.11.2 (tags/v3.11.2:878ead1, Feb  7 2023, 16:38:35) [MSC v.1934 64 bit (AMD64)]
```

### OS / Environment

Win10

**Comment by DanielNoord** [comment]
I have never seen this pattern. How common is this?

**Comment by jacobtylerwalls** [comment]
I have seen the get_text alias as `_`.

I think we should accept this on the basis that `unused-import` should observe the `dummy-variables-rgx` config argument. Right now the check only checks for a hardcoded `_`. Then this message will have parity with `unused-variable`.

Easy picking. Thanks for the report!

**Comment by RSTdefg** [comment]
Hello, I've looked into this issue and I'd like to work on it. Thanks!
